docs/cloud-get-started.md

```diff
@@ -0,0 +1,52 @@
+---
+id: cloud-get-started
+title: Getting Started
+---
+
+## Register on the ReSolve Cloud Platform
+
+Go to the [admin.resolve.sh](https://admin.resolve.sh) page, click **Sign Up** and follow through the registration process. After registration, you will gain access to your personal admin panel.
+
+## Install the ReSolve CLI
+
+Type the following command to install the reSolve cloud CLI to your application.
+
+```bash
+yarn global add resolve-cloud
+```
+
+The next step is to use the CLI to authenticate to the reSolve Cloud Platform:
```

Comment:
to authenticate WHAT to the resolve cloud platform?

Reply:
I think, I choose a wrong word. Changed to "to log in to the reSolve Cloud Platform"